ui/src/components/pages/Namespace/index.tsx

```diff
@@ -35,6 +35,81 @@ export function Namespaces() {
       k8sEventsProps: { namespaceId },
     });
   }, [namespaceId, setSidebarProps]);
+  const defaultPipelinesData = [
```

Comment:
Move these up outside of the component, with names like DEFAULT_PIPELINES_DATA. 

In current state, it recreates them on each render which may cause unneeded re-renders

Reply:
The default data needs handleK8sEventsClick method, I can move this inside a useMemo, that would be better.

Reply:
Done!